### DISCUSSION TITLE
ETA for String field

### DISCUSSION THREAD

**Discussion by lapaniku** [milvus-io/milvus#4591] [Q&A and General discussion]
Hi, today I'm using Milvus+Redis to be able to save URIs together with vectors which is basically a workaround to store string data together with vectors. I would be really happy to get rid of the extra component (Redis). Is any ETA on the table for supporting String fields in Milvus at least for saving/retrieval?

**Comment by shiyu22** [comment]
Hey guys, you can follow this [issue](https://github.com/milvus-io/milvus/issues/1924) about string.
